tests/functional/index.js

```diff
@@ -0,0 +1,86 @@
+const { expect } = require('chai');
```

Comment:
Let's keep this one for node and browser

Reply:
Done !